wyvern/entities/feature_entity_helpers.py

```diff
@@ -22,12 +22,16 @@ def feature_map_create(*feature_data: Optional[FeatureData]) -> FeatureMap:
     for data in feature_data:
         if data is None:
             continue
-
-        if data.identifier in feature_map:
+        identifier = (
```

Comment:
what if in the model code, ppl call this:
```
self.get_feature(composite_identifier, "feature")
```
?

this will return None now right?

shall we fix the "def get_feature" and apply the similar logic to use the primary identifier their?

Reply:
Oh yes, I just did this on my local. I'll push it with the other changes

Reply:
I wonder if we should make both available? 
`self.get_feature(composite_identifier_type, composite_identifiers, "feature")`
and
`self.get_feature(primary_identifier_type, primary_identifiers, "feature")`

would return the same response except the identifier column in the df

Reply:
how can we do it without changing the `get_feature` signature?

i wonder if we should duplicate the composite identifier features for the primary identifier in the feature_map?
something like this as the feature_map:

`{"product:query": {"f1": "v1"}, "product": {"f1": "v1"}}`

downside: this won't work if there's conflict in the feature name. for exeample:
if there's another composite feature:
`{"product:user": {"f1": "v2"}}`

Reply:
okay. yes let's make both available but without changing the signature of the `get_feature` function